section11/numberformat_proposed.html

```diff
@@ -143,11 +162,19 @@ <h1>PartitionNumberPattern ( _numberFormat_, _x_ )</h1>
       </p>
 
       <emu-alg>
-        1. If _x_ is not *NaN* and _x_ < 0, then
-          1. Let _x_ be -_x_.
-          1. Let _pattern_ be _numberFormat_.[[NegativePattern]].
+        1. Let _exponent_ be 0.
+        1. If _x_ is *NaN*, then
+          1. Let _n_ be an ILD String value indicating the *NaN* value.
```

Comment:
I don't have a suggestion for changes, but I'm wondering, do you have a good sense of when this spec uses ILD/ILND strings, and when it uses data explicitly in the locale object?

Reply:
My sense for ILD versus data is that ILD tends to be a small and specific string, whereas data can have many more dependencies.  For example, you could make a specific list of the ILD strings that an implementation needs to provide, but for data, the the branching factor is much larger and you can't make a specific list.

These two strings, NaN and Infinity, were already in the spec; I just moved them up a bit.

Reply:
BTW, why was this line moved up to this location, from its previous lower spot?

Reply:
I moved the calculation for _n_ and _x_ in the normal case (non-NaN-and-Infinity) up here because the new variable _exponent_ is referenced in the outer scope.  Then it looked strange that the calculation of _n_ in the special cases (NaN and Infinity) was in a different logical place than the calculation of _n_ in the normal case, so I moved those two lines up here.